sdk/src/Services/S3/Generated/Model/DeleteBucketAnalyticsConfigurationRequest.cs

```diff
@@ -0,0 +1,148 @@
+    /// 
+    /// </para>
+    ///  </li> <li> 
+    /// <para>
+    ///  <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBucketAnalyticsConfigurations.html">ListBucketAnalyticsConfigurations</a>
+    /// 
+    /// </para>
+    ///  </li> <li> 
+    /// <para>
+    ///  <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketAnalyticsConfiguration.html">PutBucketAnalyticsConfiguration</a>
+    /// 
+    /// </para>
+    ///  </li> </ul> <important> 
+    /// <para>
+    /// You must URL encode any signed header values that contain spaces. For example, if
+    /// your header value is <c>my file.txt</c>, containing two spaces after <c>my</c>, you
+    /// must URL encode this value to <c>my%20%20file.txt</c>.
+    /// </para>
+    ///  </important>
+    /// </summary>
+    public partial class DeleteBucketAnalyticsConfigurationRequest : AmazonWebServiceRequest
+    {
+        private string _analyticsId;
+        private string _bucketName;
+        private string _expectedBucketOwner;
+
+        /// <summary>
+        /// Gets and sets the property AnalyticsId. 
+        /// <para>
+        /// The ID that identifies the analytics configuration.
+        /// </para>
+        /// </summary>
+        [AWSProperty(Required=true)]
+        public string AnalyticsId
+        {
+            get { return this._analyticsId; }
+            set { this._analyticsId = value; }
+        }
+
+        // Check to see if AnalyticsId property is set
+        internal bool IsSetAnalyticsId()
+        {
+            return this._analyticsId != null;
+        }
+
+        /// <summary>
+        /// Gets and sets the property BucketName. 
+        /// <para>
+        /// The name of the bucket from which an analytics configuration is deleted.
+        /// </para>
+        /// </summary>
+        [AWSProperty(Required=true)]
+        public string BucketName
+        {
+            get { return this._bucketName; }
+            set { this._bucketName = value; }
+        }
+
+        // Check to see if BucketName property is set
+        internal bool IsSetBucketName()
+        {
+            return this._bucketName != null;
+        }
+
+        /// <summary>
+        /// Gets and sets the property ExpectedBucketOwner. 
+        /// <para>
+        /// The account ID of the expected bucket owner. If the account ID that you provide does
+        /// not match the actual owner of the bucket, the request fails with the HTTP status code
+        /// <c>403 Forbidden</c> (access denied).
+        /// </para>
+        /// </summary>
+        public string ExpectedBucketOwner
+        {
+            get { return this._expectedBucketOwner; }
+            set { this._expectedBucketOwner = value; }
+        }
+
+        // Check to see if ExpectedBucketOwner property is set
+        internal bool IsSetExpectedBucketOwner()
+        {
+            return this._expectedBucketOwner != null;
```

Comment:
The customization had `return !String.IsNullOrEmpty(this._expectedBucketOwner);` but this uses `!= null` instead.